docs/conf.py

```diff
@@ -39,8 +38,8 @@ def setup(app):
 sys.path.insert(0, os.path.abspath('exts/'))
 
 # -- General configuration ------------------------------------------------
-conf.read([os.path.join(os.path.dirname(__file__), '..', 'setup.cfg')])
-setup_cfg = dict(conf.items('metadata'))
+conf = toml.load('../pyproject.toml')
+setup_cfg = conf['project']
```

Comment:
Sphinx does not currently read `pyproject.toml` natively, so we can use the `toml` library here to read the configuration directly